cfgov/jinja2/v1/about-us/index.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'layout-2-1-bleedbar.html' %}
 
 {% block title -%}
-    About us
+    About us | Consumer Financial Protection Bureau
 {%- endblock %}
```

Comment:
I'm confused about all of these where the title is hardcoded. It would always render "About us" so why the or statement here?  And, shouldn't it just be calling super() to make use of the logic added in `base.html` ?